test/create-node-middleware.test.ts

```diff
@@ -61,7 +83,9 @@ describe("createNodeMiddleware", () => {
 
     const body = JSON.stringify(pushEvent);
 
-    await got.post(`http://127.0.0.1:${port}`, {
+    // @ts-ignore
```

Comment:
See https://github.com/DefinitelyTyped/DefinitelyTyped/issues/60924
For now, you could use the DOM libs

Reply:
done